platforms/java/com/area9innovation/flow/Native.java

```diff
@@ -66,7 +66,12 @@ public Native() {
 		}
 	}
 
-	public final Object println(Object arg) {
+	public void initialize() {
+		Integer emptyList_id = FlowRuntime.struct_ids.get("EmptyList");
+		emptyList = FlowRuntime.struct_prototypes[emptyList_id];
```

Comment:
Why `EmptyList` is required? The flow program may not use Lists.

Reply:
EmptyList is used in tailList native

Reply:
If  some my flow program do not use Lists, then `emptyList_id` will be -1, right?

Reply:
I added a check on null value of emptyList_id.